### ISSUE TITLE
Trans component not updating

### ISSUE THREAD

**Issue by fabien** [vinissimus/next-translate#562]
The following will not dynamically update when the page locale changes, while the `useTranslation` hook version does.

The initially loaded version remains in Trans component:

```javascript
<Trans i18nKey="common:modal.accept" /> vs. {t('common:modal.accept')}
```

Whereas the following workaround displays the correct translation for Trans:

```javascript
<Trans i18nKey="common:modal.accept" components={[]} /> vs. {t('common:modal.accept')}
```

**Comment by aralroca** [comment]
I think adding the lang as dependencies here would solve the problem:

https://github.com/vinissimus/next-translate/blob/master/src/Trans.tsx#L83

**Comment by aralroca** [comment]
@fabien this will be released on 1.0.6 but I pre-released the fix under [1.0.6-canary.1](https://github.com/vinissimus/next-translate/releases/tag/1.0.6-canary.1) if you want to start using it. Can you confirm that is working for you? Thanks!

**Comment by fabien** [comment]
@aralroca thanks for the quick fix, it works perfectly now!
